### PR TITLE
Convert to md

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: src/site/markdown/download.md.vm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+${project.name} is distributed under the [Apache License, version 2\.0](http://www.apache.org/licenses/)\.
+
+We **strongly** encourage our users to configure a Maven repository mirror closer to their location, please read [How to Use Mirrors for Repositories](./guides/mini/guide-mirror-settings.html)\.
+
+<a id="mirror"></a>
+
+${esc.h}${esc.h} Mirror
+
+\[if-any logo\]
```
Comment (kwin) [inline src/site/markdown/download.md.vm:44]: @slawekjaranowski Is this still supposed to be used like this? I am not sure which placeholders are replaced by the CGI (documentation at https://infra.apache.org/release-download-pages.html#custom is very sparse). Mirrors are gone nowadays...

Comment (kwin) [reply src/site/markdown/download.md.vm:44]: There is just very few placeholders available for use in the template: https://maven.apache.org/skins/maven-fluido-skin/download.cgi?asjson=true, the templating language is described in https://github.com/gstein/ezt/blob/wiki/Syntax.md.

Comment (slawekjaranowski) [reply src/site/markdown/download.md.vm:44]: First I would like to update download page like in https://github.com/apache/maven-changes-plugin/pull/68
we will not a special cgi for it

Comment (slawekjaranowski) [reply src/site/markdown/download.md.vm:44]: https://issues.apache.org/jira/browse/MSKINS-240

Comment (kwin) [reply src/site/markdown/download.md.vm]: As a globally provided template is some more efforted I refactored now the download page to a more simple version (no longer mentioning any mirrors). However it still relies on the placeholder mechanism of the underlying [closer.lua script](https://github.com/apache/infrastructure-p6/blob/production/modules/closer_cgi/files/closer.lua) due to reasons outlined at https://issues.apache.org/jira/browse/MSKINS-240?focusedCommentId=17902720&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17902720

File: src/site/markdown/download.md.vm [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+This is the current stable version of ${project.name}\.
+
+|  |Link|Checksum|Signature|
+|---|---|---|---|
+${project.name} ${project.version} \(Source zip\)|[maven/skins/${project.artifactId}\-${project.version}\-source\-release\.zip](\[preferred\]/maven/skins/${project.artifactId}-${project.version}-source-release.zip)|[maven/skins/${project.artifactId}\-${project.version}\-source\-release\.zip\.sha512](https://www.apache.org/dist/maven/skins/${project.artifactId}-${project.version}-source-release.zip.sha512)|[maven/skins/${project.artifactId}\-${project.version}\-source\-release\.zip\.asc](https://www.apache.org/dist/maven/skins/${project.artifactId}-${project.version}-source-release.zip.asc)|
+
+${esc.h}${esc.h} Previous Versions
+
+Older non\-recommended releases can be found on our [archive site](http://archive.apache.org/dist/maven/skins/)\.
```
Comment (slawekjaranowski) [inline src/site/markdown/download.md.vm:45]: https

**Review by slawekjaranowski — APPROVED** [review]

File: src/site/markdown/download.md.vm (hunk 2 above)
Comment (kwin) [reply src/site/markdown/download.md.vm]: Good catch, fixed now.

**Pull request merged**